test/tools/libtesteth/TestHelper.cpp

```diff
@@ -181,6 +181,66 @@ set<eth::Network> const& getNetworks()
     return networks;
 }
 
+/// translate network names in expect section field
+/// >Homestead to EIP150, EIP158, Byzantium, ...
+/// <=Homestead to Frontier, Homestead
+set<string> translateNetworks(set<string> const& _networks)
+{
+    // construct vector with test network names in a right order (from Frontier to Homestead ... to
+    // Constantinople)
+    vector<string> forks;
+    for (auto const& net : getNetworks())
+        forks.push_back(test::netIdToString(net));
+
+    set<string> out;
+    for (auto const& net : _networks)
+    {
```

Comment:
Optional: you can factor out
```
set<string> translateNetwork(string const& _network)
```
which does only one iteration of this loop.

Reply:
what you mean by "factor out"?

Reply:
To factor out a piece of code is to move it into the body of a new function.

Reply:
if the subfunction (in this case cycle body) is not used anywhere else. I prefere to keep it at one place. unless it is too big.

Reply:
OK.